components/prize_pool/commons/prize_pool_placement.lua

```diff
@@ -32,7 +32,7 @@ local SPECIAL_SCORES = {'W', 'FF' , 'L', 'DQ', 'D'}
 
 local _tbd_index = 0
 
---- @class Placement
+--- @class PrizePoolPlacement
```

Comment:
does that need adjusts in other modules?

Reply:
Not that I could find